lib/Cro/HTTP/Client.pm6

```diff
@@ -731,4 +751,14 @@ class Cro::HTTP::Client {
     method choose-connector($secure) {
         $secure ?? Cro::TLS::Connector !! Cro::TCP::Connector
     }
+
+    multi method set-http-proxy(Cro::Uri $!http-proxy) {}
+    multi method set-http-proxy(Str $http-proxy) {
+        $!http-proxy = Cro::Uri::HTTP.parse($http-proxy);
+    }
+
+    multi method set-https-proxy(Cro::Uri $!https-proxy) {}
+    multi method set-https-proxy(Str $https-proxy) {
+        $!https-proxy = Cro::Uri::HTTP.parse($https-proxy);
+    }
```

Comment:
I'm not sure we really need these; we only take all the other client options as constructor parameters, and I think that's enough.